.gitignore

```diff
@@ -69,4 +69,4 @@ caketools/
 *.binlog
 .ionide/**
 /.nuspec
-XamarinFormsVersionFile.txt
+.XamarinFormsVersionFile.txt
```

Comment:
@rmarinho yea

Otherwise git will try to check the files in

![image](https://user-images.githubusercontent.com/5375137/67230942-89750800-f3fb-11e9-9314-f4c159bd4df0.png)